ReportPortal.Tests/ReportPortal.E2E.API.Business/Models/Responses/ErrorResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Text.Json.Serialization;
+
+namespace ReportPortal.E2E.API.Business.Models.Responses
+{
+    public class ErrorResponse
+    {
+        [JsonPropertyName("errorCode")]
+        public int ErrorCode { get; set; }
+
```

Comment:
This is not a remark. It's more of a question. Is it justified to use jsonproperty in models? can't it be confusing in the future?

Reply:
JsonPropertyName is used to get exactly name in deserialization

In case you would like to know something or need additional explanation please fill free to connect with directly to avoid of creation of unnecessary comments

Reply:
Discussed